scripts/dev/format-stackdump.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function format() {
+sed -e "s#$GOPATH#\$GOPATH#g" -e 's/$GOPATH/\
+$GOPATH/g' -e 's/<body>/\
+<body>\
+/g' -e 's#</body>#\
+</body>#' -e 's# /usr/lib/go/src#\
+/usr/lib/go/src#g'
+}
+
+if [ -n "$1" ]; then
+	if [ "$1" == "-h" -o "$1" == "--help" ]; then
+		echo "$0: reads an unformatted stackdump, and formats it"
+		echo "replaces your GOPATH with the string \#GOPATH and introduces newlines where appropriate"
+		echo "usage:"
+		echo "$0 --help, -h     this message"
+		echo "$0 <filename>     reads input from file"
+		echo "$0                reads input from stdin"
+		exit 0
+	elif [ -r "$1" ]; then
+		format <<< "$1"
```

Comment:
Using a filename didn't seem to work in the docker container I was testing with (bash version `4.3.42`, possibly too old).

Reply:
Weird. bash v4 is very recent, all things considered.
per https://tiswww.case.edu/php/chet/bash/NEWS, the <<< redirection operator was introduced in bash-2.05b
maybe something else went wrong in your container?

Reply:
`cat "$1" | format` did work, at the cost of an additional process

Reply:
It seems that here-strings are for string literals, so `format <<< "$1"` passes the literal filename as input to `format` not the contents of the file. Unless I'm missing something.

Reply:
oops yes i mixed up `<<<` with `<`